Exception/ExceptionInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Opensoft\StorageBundle\Exception;
+
+/**
+ * @author Evgeny Sapozhnikov <evgeny.sapozhnikov@opensoftdev.ru>
+ */
+interface ExceptionInterface
```

Comment:
I would prefer throws custom Exception. Looks like we apply constant from one type to another type.